module_message.lua

```diff
@@ -462,9 +483,14 @@ function Module:OnMessageCreate(message)
 	local config = self:GetConfig(message.guild)
 	local reply = config.Replies[message.content]
 	if (reply) then
-		local success, err = message:reply(reply)
+		reply = ShallowCopy(reply)
+		local deleteInvokation = RemoveTableKey(reply, "deleteInvokation")
+	 	local success, err = message:reply(reply)
+		
 		if (not success) then
 			self:LogError(message.guild, "Failed to reply to %s: %s", message.content, err)
+		elseif (deleteInvokation == nil or deleteInvokation and config.DeleteInvokation) then
```

Comment:
This condition is not clear

Reply:
This is the cleanest way I've found to make a "ternary

Reply:
`deleteInvokation ~= nil and deleteInvokation or config.DeleteInvokation` is the idiomatic way

Reply:
Also, since `nil` is evalutated as `false`, this can be simplified to `deleteInvokation or config.DeleteInvokation`

Reply:
oui sauf que si `deleteInvokation` est a `false` le message ne doit pas être supprimé

Reply:
yes except that if `deleteInvokation` is set to `false` the message should not be deleted

Reply:
Oh yeah, then the idiomatic way of writing it would be `deleteInvokation == nil and config.DeleteInvokation or deleteInvokation`

Reply:
no, `deleteInvokation == nil or deleteInvokation and config.DeleteInvokation` works correctly.
`deleteInvokation == nil and config.DeleteInvokation or deleteInvokation` does the opposite (`config.DeleteInvokation` should not be a priority to `deleteInvokation`)

Reply:
yes :+1: